README.md

```diff
@@ -35,87 +48,47 @@ npm start -- --client-id=main
-
-Some plugins require TLS to function properly (such as hardware signing
-support).
-
-### About Docker Environment
-(To learn how to use bpanel for existing bcoin nodes on any network,
-read about [configuring bPanel](#configuration) below).
-
-The following will generally just be used for quick start, testing,
-and development purposes (though it could be used in production
-with some modification).
-
-To spin up a webapp, reverse-proxy, server, a bcoin node on regtest and a wallet server
-with [multisig support](https://github.com/bcoin-org/bmultisig),
-clone this repo and from the cloned directory do the following:
+Read more about how to connect a local instance of bPanel to a bcoin node running in docker
+[here](#local-bpanel-dev-using-bcoin-docker-container).
 
-1. Run `npm install` to install dependencies and generate
-necessary constants.
-1. Run `docker-compose up -d` to start everything.
-1. Navigate to [localhost:5000](http://localhost:5000) to see your webapp.
-1. Or navigate to [https://localhost](https://localhost) to use TLS - you will have to
-choose to trust the certificate
+## Managing Plugins
 
-Requests to the `/bcoin` will get forwarded to your bcoin node.
-Requests to the `/bwallet` endpoint will get forwarded to the wallet server
+Plugin management is handled in a `js` file located by default in the `.bpanel` directory in your
+system's home directory (this location can be manually changed with the `BPANEL_PREFIX` env variable or
+`--prefix` option at run time.
 
-For local development, you run just the bcoin docker container (`docker-compose
-up -d bcoin`) and then `npm run start:dev -- --client-id=_docker`
-(or `npm run start:poll -- --client-id=_docker` for Mac since
-webpack's watch can behave strangely on macOS) to run bPanel and its server
-from your local box.
-
-Note that the `--client-id` argument tells bPanel which client config
-you want to use. `_docker` is the name of the config created by the bcoin
-service.
-
-## Updating Plugins
-Plugin management is handled in a `js` file located in the `.bpanel` directory in your
-system's home directory.
-
-bPanel comes pre-installed with a default theme called [`Genesis Theme`](https://github.com/bpanel-org/genesis-theme),
-that bundles together a set of useful starter plugins and a custom skin called bMenace.
+bPanel comes pre-installed with a default theme called [`Genesis Theme`](https://github.com/bpanel-org/genesis-theme)
+that bundles together a set of useful starter plugins and a custom skin called [bMenace](https://github.com/bpanel-org/bmenace-theme).
 If you want, you can disable the Genesis Theme by removing it from the list in `config.js`,
 but if you want to keep using _some_ of the plugins from the theme, feel free to add
 them individually to your config!
 
+### Installing Plugins
+
 To install plugins, simply add the name as a string to the `plugins` array in `~/.bpanel/config.js`.
 Make sure to match the name to the package name on npm
```

Comment:
example might be helpful here to show punctuation, formatting:

>   plugins: ['@bpanel/bmoon-theme', '@bpanel/publish-data']

Reply:
Added an example. FWIW, the formatting/punctuation around the `@` for example is still strictly the plugin name. i.e. whatever you have to type for `npm install` is what should be written in the array. Our plugins just happen to be scoped on npm under `@bpanel`